src/model/persistence.ts

```diff
@@ -26,6 +28,10 @@ export const initialize = async () => {
     }
     if (data.config) {
       config.set(data.config)
+
+      // Check config for "startInFullScreen"
+      // console.log('start in fullscreen:', get(config).startInFullScreen);
+      if (get(config).startInFullScreen) { ui.isFullScreen.update(isFS => !isFS) }
```

Comment:
I think I see a potential bug here.

Looking at the code, this is what I see:
If `config.startInFullScreen` is `true` then flip the value of `ui.isFullScreen`.

What I expect to see:
Set `ui.isFullScreen` to the value of `config.startInFullScreen`.

So after thinking this through, I would **not** expect to see a bug right now.

But that's because the default value of `ui.isFullScreen` is false, _SO_ then **if** `config.startInFullScreen` is true, then `ui.isFullScreen` is flipped from false to true. ✅

However, if later we changed the default value of `ui.isFullScreen` to true, then `config.startInFullScreen` is true, it would get flipped to false. 🐞

---

**Bigger context:**
Now this specific situation might never actually cause a problem.
But, to use Jiujitsu as an example, it's kind of like keeping your arms close to your body. You might not get punished this time, but it could lead to getting armbar-ed. (Especially if you do it enough)

So we can avoid any future problems by changing this line to:
`ui.isFullScreen.set(get(config).startInFullScreen) }`

Reply:
I see. So, tell me if I've got this correct.

It's better to set the value of a boolean A to its "linked" boolean B instead of "flipping" boolean A when boolean B changes. Is that right?

I think I can see how this could create bugs. Thank you for pointing this out.

Reply:
Exactly! Except, this is only happening on initialization, so it's not whenever boolean B changes. Happy to discuss to clarify!